recipes/tibanna/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "tibanna" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "aa63462bb2e114442baca9cb1ee25980c3f03356d2c280ed7eabf694c79061db"
+
+build:
+  number: 0
+  entry_points:
+    - tibanna = tibanna.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [not py36]
+
+requirements:
+  host:
+    - benchmark-4dn ==0.5.6
+    - boto3 >=1.9.0
+    - botocore >=1.12.1
+    - pip
+    - python
```

Comment:
```suggestion
    - python >=3.7,<3.6 
```

Reply:
You do not want to build a packaged just for ``3.6``?

Reply:
Nope, it was just part of trying out why it wants to pull futures. We once saw a built where it succeeded on py36... Noarch with python restriction should be fine, and was my initial suggestion as well.